apps/pie-docs/src/components/button/style/destructive-colour-tokens.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
yeah agree it's worth discussing to avoid excess overhead - I'll ask Naz about it when she's back and maybe we can have a meeting to discuss how best to implement. Thanks for bringing up 👍

Reply:
That'd be great. Just want to make sure we're not making a tonne of work for ourselves, especially as we'd be listing these out across every component.

Reply:
we decided to hide the style tab for now